### PR TITLE
Further polish to the powerdown indicator

### PR DESCRIPTION
**Pull request by abcdefg30**
The blocked animation looked weird (too small). I could use the green color instead of the light blue if wished. The next thing is that the powerdown indicator was below the offline building.
ra2powerdown

### CODE REVIEW

File: cursors.yaml [hunk 1]
```diff
@@ -273,7 +273,7 @@ Cursors:
 			Start: 73
 			Length: 5
 		powerdown-blocked:
-			Start: 52
+			Start: 345
```
Comment (Mailaender) [inline cursors.yaml]: I think it may be worse repeating `339` here without animation, or `342` to show a greyed out version. This looks like an unrelated movement block cursor which is a bit irritating. Feels like I am not in powerdown mode.

**Comment by abcdefg30** [comment]
Updated. Using the sequence index `342` now.
